### DISCUSSION TITLE
Any ways to help localize the project?

### DISCUSSION THREAD

**Discussion by faq0** [MicaForEveryone/MicaForEveryone#148] [Q&A] [answered]
This is a really great project, but it only support English now. Is there a way to help localize the project?

**Comment by xmine64** [comment, marked as answer]
It supports other languages as well but some how they doesn't appear in MSIX version. If you want to localize you can translate [this](https://github.com/MicaForEveryone/MicaForEveryone/blob/master/MicaForEveryone.UI/Strings/en/Resources.resw) file.

**Reply by xmine64** [reply]
Available translations:
* German
* English
* Spanish
* Persian
* French
* Italian
* Japenese
* Korean
* Russian
* Chinese
